fuzzinator/mongo_driver.py

```diff
@@ -67,6 +70,8 @@ def update_issue(self, issue, _set):
         self._db.fuzzinator_issues.update_one({'id': issue['id'], 'sut': sut}, {'$set': _set})
 
     def remove_issue_by_id(self, _id):
+        if not isinstance(id, ObjectId):
+            id = ObjectId(id)
```

Comment:
The `id` checked and transformed here is not the `_id` that's received as the function argument and then passed on to `delete_one`. Right now, this addition is a no-op. (Should check and transform `_id`.)

Reply:
Done.